src/update.ts

```diff
@@ -4,34 +4,24 @@ import { ColorifyConstants } from './api/constants/Colors'
 import { Messages } from './lib/constants/Messages'
 
 import * as pkg from '../package.json'
+import { getDistTag, getSimpleVersion } from './modules/utils'
 
 export function updateNotify() {
-  const notifier = updateNotifier({ pkg, updateCheckInterval: 1000 * 60 * 60 * 1 })
-  if (notifier.update && notifier.update.latest !== pkg.version) {
-    const oldVersion = notifier.update.current
+  const distTag = getDistTag(pkg.version)
+  const notifier = updateNotifier({ pkg, distTag, updateCheckInterval: 1000 * 60 * 60 * 1 })
```

Comment:
Update `1000 * 60 * 60 * 1` with a more verbose variable name. I'm not sure what's the unit of measure, but let's say this means one hour:
```
const ONE_HOUR = 1000 * 60 * 60 * 1
...
const notifier = updateNotifier({ pkg, distTag, updateCheckInterval: ONE_HOUR })
```